src/module.ts

```diff
@@ -139,6 +139,23 @@ export const plugin = new PanelPlugin<PanelOptions>(ImagePanel)
         defaultValue: DEFAULT_OPTIONS.zoomType,
         category: ['Toolbar'],
         showIf: (options: PanelOptions) => options.toolbar && options.buttons.includes(ButtonType.ZOOM),
+      })
+      .addNumberInput({
+        path: 'autoPlayInterval',
+        name: 'Auto play Interval',
+        description: 'Set interval for auto play in sec. If not specified, 5 sec. set by default',
+        category: ['Toolbar'],
+        showIf: (options: PanelOptions) => options.toolbar && options.buttons.includes(ButtonType.AUTOPLAY),
+      })
+      .addRadio({
+        path: 'sliderAutoPlayInfinity',
```

Comment:
Can we update it to `autoPlayInfinity` similar to `autoPlayInterval`? I think `slider` is redundant.